datasets/kwaliteitsmonitor/wonenVerblijfsobject/v1.0.0.json

```diff
@@ -70,14 +74,14 @@
         "description": "De fase van de levenscyclus van een verblijfobject waarin de betreffende verblijfobject zich bevindt"
       },
       "bagStatusCode": {
-        "type": "string",
+        "type": "integer",
         "description": "Code behorend bij status verblijfsobject"
       },
       "pndIdentificatie": {
-        "type": "integer",
+        "type": "string",
         "description": "Unieke identificatie van het pand"
       },
-      "pndOorspronkelijkbouwjaar": {
+      "pndOorspronkelijkBouwjaar": {
```

Comment:
"pandOorspronkelijkBouwjaar" of "oorspronkelijkBouwjaarPand"

Reply:
pndOorspronkelijkBouwjaar, het staat goed